src/Microsoft.DotNet.Interactive.Http.Parsing/Parsing/HttpRequestParser.cs

```diff
@@ -194,6 +194,10 @@ private HttpVariableDeclarationNode ParseVariableDeclaration()
                     {
                         ConsumeCurrentTokenInto(node);
                     }
+                    else if(CurrentToken is { Text: "_" })
+                    {
+                        ConsumeCurrentTokenInto(node);
```

Comment:
This addresses the variable declarations, but don't we also need to handle this inside embedded expressions (i.e. when parsing variable names that appear as part of expressions)?

Reply:
Just created a test for this but we don't have to check in embedded expressions because the way it set up currently is that anything inside of the curly braces will be parsed as the expression.